migrations/000041_AddTravelerFederation.up.sql

```diff
@@ -0,0 +1,34 @@
+-- Copyright 2020 Google LLC
+--
+-- Licensed under the Apache License, Version 2.0 (the "License");
+-- you may not use this file except in compliance with the License.
+-- You may obtain a copy of the License at
+--
+--      http://www.apache.org/licenses/LICENSE-2.0
+--
+-- Unless required by applicable law or agreed to in writing, software
+-- distributed under the License is distributed on an "AS IS" BASIS,
+-- WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+-- See the License for the specific language governing permissions and
+-- limitations under the License.
+
+BEGIN;
+
+ALTER TABLE federationinquery
+    ADD COLUMN only_local_provenance BOOL DEFAULT TRUE,
```

Comment:
I think we need to index some of these. Particularly the cursors and timestamps. Not sure if we get a perf gain by indexing the bools.

Reply:
we never search on them - federationinquery always loaded by ID or select *